package/skelcd-control-openSUSE.spec

```diff
@@ -132,7 +132,7 @@ install -m 644 control/${CONTROL_FILE} $RPM_BUILD_ROOT%{?skelcdpath}/CD1/control
     sed -i -e "s,http://download.opensuse.org/source/,http://download.opensuse.org/ports/$ports_arch/source/," %{buildroot}%{?skelcdpath}/CD1/control.xml
     sed -i -e "s,http://download.opensuse.org/update/tumbleweed/,http://download.opensuse.org/ports/$ports_arch/update/tumbleweed/," %{buildroot}%{?skelcdpath}/CD1/control.xml
     #we parse out non existing non-oss repo for ports
-    xsltproc -o %{buildroot}%{?skelcdpath}/CD1/control.xml control/nonoss.xsl %{buildroot}%{?skelcdpath}/CD1/control.xml/
+    xsltproc -o %{buildroot}%{?skelcdpath}/CD1/control_ports.xml control/nonoss.xsl %{buildroot}%{?skelcdpath}/CD1/control.xml
```

Comment:
if you change it this way, then mv below is wrong as it moves control_ports which overwrites xslt proc result. So please remove mv below or set as last parameter control_ports.xml instead of control.xml

Thanks for fix. I will merge it afterwards.

Reply:
I do not understand :( I thought the lines was to replace the content of control.xml by output of xsltproc call (that is control_ports.xml) 
do you suggest to keep two xml files ? (control.xml and control_ports.xml)

Reply:
@jreidinger  am I wrong ?

Reply:
@michelmno problem here is that you have 
```
xsltproc -o %{buildroot}%{?skelcdpath}/CD1/control_ports.xml control/nonoss.xsl %{buildroot}%{?skelcdpath}/CD1/control.xml
```
this will create file control.xml
and then next line is
```
mv %{buildroot}%{?skelcdpath}/CD1/control{_ports,}.xml
```
which moves control_ports to control.xml so basically you overwrite any changes done by first line as it transform control_port.xml into control.xml

Reply:
@jreidinger , I ran manually in osc chroot env to confirm that the
-o %{buildroot}%{?skelcdpath}/CD1/control_ports.xml  is the OUTPUT of xsltproc call and
    %{buildroot}%{?skelcdpath}/CD1/control.xml input is not modified by first line.

So next line moves control_ports.xml (OUTPUT of xsltproc) to control.xml  as expected.

Reply:
ah, now I get it. OK, so makes sense. I will merge it.